src/bin/cql-stress-cassandra-stress/java_generate/values/inet.rs

```diff
@@ -0,0 +1,29 @@
+use std::net::{IpAddr, Ipv4Addr};
+
+use scylla::frame::response::result::CqlValue;
+
+use crate::java_generate::distribution::Distribution;
+
+use super::{ValueGenerator, ValueGeneratorFactory};
+
+#[derive(Default)]
+pub struct Inet;
+
+impl ValueGenerator for Inet {
+    fn generate(
+        &mut self,
+        identity_distribution: &mut dyn Distribution,
+        _size_distribution: &mut dyn Distribution,
+    ) -> CqlValue {
+        let v = (identity_distribution.next_i64() as i32).to_be_bytes();
+        CqlValue::Inet(IpAddr::V4(Ipv4Addr::new(v[0], v[1], v[2], v[3])))
+    }
+}
```

Comment:
A unit test is missing here, and I claim that this code yields different results than the Java one:
```java
int val = (int) identityDistribution.next();
buf[0] = (byte)(val >>> 24);
buf[1] = (byte)(val >>> 16);
buf[2] = (byte)(val >>> 8);
buf[3] = (byte)val;
return InetAddress.getByAddress(buf);
```
`buf` in Java stores val in LittleEndian, whereas `v` in Rust stores it in BigEndian.

Reply:
Nit: you can use this impl: 
impl [From](https://doc.rust-lang.org/std/convert/trait.From.html)<[[u8](https://doc.rust-lang.org/std/primitive.u8.html); [4](https://doc.rust-lang.org/std/primitive.array.html)]> for [Ipv4Addr](https://doc.rust-lang.org/std/net/struct.Ipv4Addr.html)

Reply:
If I'm not mistaken and there is a difference in results, how have you tested your impls?

Reply:
You can run the test from CI:
```bash
# Setup a cluster
docker compose -f docker/scylla_test/compose.yml up -d --wait

# Run the test
pytest -s ./tools/cassandra_stress_ci.py -k test_user_inet_type

# You can examine the tables:
cqlsh
> SELECT * FROM cs_inet_keyspace.test_table ;
> SELECT * FROM cqlstress_inet_keyspace.test_table ;
```

Reply:
You're right, reading it now I have no idea why I reasoned it's LE in Java.